wasmedge-ggml-belle-chat/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "wasmedge-ggml-belle-chat"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+chat-prompts = "0.1"
+endpoints = "0.1"
+wasi-nn = { git = "https://github.com/second-state/wasmedge-wasi-nn", branch = "dm4/ggml" }
```

Comment:
@dm4 
Please check if we are still on this branch?

Reply:
Please use the official `ggml` branch instead. Thank you!

Reply:
Fixed. Thanks!